openlibrary/core/wikidata.py

```diff
@@ -0,0 +1,147 @@
+"""
+The purpose of this file is to:
+1. Interact with the Wikidata API
+2. Store the results
+3. Make the results easy to access from other files
+"""
+
+import requests
+from dataclasses import dataclass
+from openlibrary.core.helpers import days_since
+
+from datetime import datetime
+import json
+from openlibrary.core import db
+
+WIKIDATA_API_URL = 'https://www.wikidata.org/w/rest.php/wikibase/v0/entities/items/'
+WIKIDATA_CACHE_TTL_DAYS = 30
+
+
+@dataclass
+class WikidataEntity:
+    """
+    This is the model of the api response from WikiData plus the updated field
+    https://www.wikidata.org/wiki/Wikidata:REST_API
+    """
+
+    id: str
+    type: str
+    labels: dict
+    descriptions: dict
+    aliases: dict
+    statements: dict
+    sitelinks: dict
+    updated: datetime  # This is when we fetched the data, not when the entity was changed in Wikidata
```

Comment:
Because this is the only field that is not actually returned by https://www.wikidata.org/w/rest.php/wikibase/v0/entities/items/Q42 , let's label it as such so that it's clear this is internal to our application

```suggestion
    _updated: datetime  # This is when we fetched the data, not when the entity was changed in Wikidata
```